test/gtest/reducecalculation.hpp

```diff
@@ -274,4 +324,6 @@ struct ReduceCalculationTest : public ::testing::TestWithParam<ReduceCalculation
     int32_t dim;
```

Comment:
consider using `uint32_t` for dim (if it can't be negative)

Reply:
solved